style.css

```diff
@@ -14,7 +14,33 @@ body {
 }
 
 .logo {
+  position: relative;
   padding: 40px 0;
+  padding-left: 44px;
+  font: bold 24px/31px pt_serif, arial, helvetica, sans-serif;
+}
+
+.logo__circle {
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+}
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
```

Comment:
Codacy found an issue: [Expected "#C8DBFB" to be "#c8dbfb" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6242822)